src/routes/homepage/Homepage.jsx

```diff
@@ -25,7 +25,7 @@ export const Homepage = () => {
   
   useEffect(() => {
     document.title = "BookStock";
-  });
+  },[]);
```

Comment:
you can make custom hook for changing document title

Reply:
sure, good suggestion!